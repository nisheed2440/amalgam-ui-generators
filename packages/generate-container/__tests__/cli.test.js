'use strict';

const cli = require('../lib/cli');

describe('@amalgam-ui/generate-container cli', () => {
  const argv = cli(process.cwd()).parse(['args']);
  console.log(argv);
  it('needs tests', () => {
    expect(true).toBe(true);
  });
});
