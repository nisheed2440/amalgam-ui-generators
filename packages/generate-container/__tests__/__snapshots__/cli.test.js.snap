// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@amalgam-ui/generate-container cli should show help info 1`] = `
"generate-container <container-name>

Create container for Amalgam UI framework.

Positionals:
  container-name  The name of the container                             [string]

Options:
  -n, --namespace  The namespace for the container package.json
  -d, --dest       The destination path for the container [default: \\"packages/\\"]
  -h, --help       Show help                                           [boolean]
  -v, --version    Show version number                                 [boolean]
"
`;

exports[`@amalgam-ui/generate-container cli should show version 1`] = `
"0.0.1-alpha.0
"
`;
